api/valor_api/backend/models.py

```diff
@@ -73,7 +73,8 @@ class GroundTruth(Base):
         ForeignKey("annotation.id"), nullable=True
     )
     label_id: Mapped[int] = mapped_column(
-        ForeignKey("label.id"), nullable=False
+        ForeignKey("label.id"),
+        nullable=True,
```

Comment:
This change confuses me as the GroundTruth table exists only to link labels. Since no other column was added, I think this should probably be reverted along with the migration change.

Reply:
@czaloom There aren't any labels for text generation groundtruths and predictions, so there is no label id. You can see in api/valor_api/backend/core/prediction.py lines 161:175 for example that I had to save the Prediction with label_id = None because there aren't any labels.

Reply:
Are there scores involved? The ground truth and prediction tables in the backend are really only label linkers and should be created if there are no labels.